shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -77,7 +78,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
 
         AccessibilityNodeInfo result = AccessibilityNodeInfo.obtain(mOwner, virtualViewId);
         result.setPackageName(mOwner.getContext().getPackageName());
-        result.setClassName("Flutter"); // Prettier than the more conventional node.getClass().getName()
+        result.setClassName("Flutter"); // TODO(goderbauer): Set proper class names
```

Comment:
Android TalkBack uses the class to determine how to behave (for an example see below where I set the class to ScrollView to get the correct scroll behavoir). And if you set the class to button for example, Android talkback will correctly announce that it is a button. There are more classes with special meaning, apparently.